File: src/test/java/com/jakobmenke/bootrestgenerator/ConfigurationTest.java
package com.jakobmenke.bootrestgenerator;

import com.jakobmenke.bootrestgenerator.utils.Configuration;
import org.junit.jupiter.api.Test;

import java.util.Properties;

import static org.junit.jupiter.api.Assertions.assertNotNull;
import static org.junit.jupiter.api.Assertions.assertNull;

class ConfigurationTest {

    @org.junit.jupiter.api.Test
    void readValidConfig() {
        Properties properties = Configuration.readConfig("config.properties");
        assertNotNull(properties);
    }
    @Test
    void readInvalidConfig() {
        Properties properties = Configuration.readConfig("ifsalf.properties");
        assertNull(properties);
    }
    @Test
    void readNullConfig() {
        Properties properties = Configuration.readConfig("ifsalf.properties");
        assertNull(properties);
    }
}

File: src/test/kotlin/com/jakobmenke/bootrestgenerator/ConfigurationTest.kt
package com.jakobmenke.bootrestgenerator

import com.jakobmenke.bootrestgenerator.utils.Configuration
import org.junit.jupiter.api.Test
import kotlin.test.assertNotNull
import kotlin.test.assertNull

class ConfigurationTest {
    @Test
    fun readValidConfig() {
        val properties = Configuration.readConfig("config.properties")
        assertNotNull(properties)
    }

    @Test
    fun readInvalidConfig() {
        val properties = Configuration.readConfig("ifsalf.properties")
        assertNull(properties)
    }

    @Test
    fun readNullConfig() {
        val properties = Configuration.readConfig("ifsalf.properties")
        assertNull(properties)
    }
}
